doozer/doozerlib/cli/scan_fips.py

```diff
@@ -96,9 +97,18 @@ def get_all_latest_builds(self):
         for tag in brew_tags:
             builds += self.get_tagged_latest(tag=tag)
 
-        nvrs = []
+        # Find the latest build nvr per image across all tags
+        latest_build_map = {}
         for build in builds:
-            nvrs.append(build["nvr"])
+            image_name = build["name"]
```

Comment:
I think `build["package_name"]` could be a better key

Reply:
Thank you, should be fixed now